components/eam/src/chemistry/utils/iac_coupled_fields.F90

```diff
@@ -0,0 +1,385 @@
+module iac_coupled_fields
```

Comment:
Shouldn't this file be in eam/src/utils ?  Its not used only by the chemistry.

Reply:
Moved file to `eam/src/utils/`.